formatter/fmt

```diff
@@ -65,7 +65,7 @@ else
   npx eslint --fix "app/assets/javascripts/**/*.ts"
   cd ../browser-test
   npm install --silent
-  npx eslint --fix
+  npx eslint --fix "src/**/*.ts"
```

Comment:
This seem unrelated to this PR, should it be here?

Reply:
My first PR attempt failed due to lint error and exposed the fact that formatter/format currently doesn't run linter properly on browser-test. Here I'm fixing it. I decided not to pull it out to separate PR as the change is tiny and overhead of creating PR is bigger.

Reply:
Please include in the description then as otherwise it looks like an error.